e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts

```diff
@@ -10,31 +11,41 @@ import { Common, setupBrowser } from "../../../utils/common";
 import { UIhelper } from "../../../utils/ui-helper";
 import fs from "fs/promises";
 import { RbacPo } from "../../../support/pageObjects/rbac-po";
+import { RhdhAuthApiHack } from "../../../support/api/rhdh-auth-api-hack";
+import RhdhRbacApi from "../../../support/api/rbac-api";
+import { RbacConstants } from "../../../data/rbac-constants";
+import { Policy } from "../../../support/api/rbac-api-structures";
+
+/*
+    Note that:
+    The policies generated from a policy.csv or ConfigMap file cannot be edited or deleted using the Developer Hub Web UI. 
+    https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/authorization/managing-authorizations-by-using-the-web-ui#proc-rbac-ui-edit-role_title-authorization
+*/
+test.describe.serial("Test RBAC", () => {
+  test.describe
+    .serial("Test RBAC plugin: load permission policies and conditions from files", () => {
+    test.beforeEach(async ({ page }) => {
+      await new Common(page).loginAsKeycloakUser();
```

Comment:
I am not aware of if there is any reason but I guess performing login after each test will increase the test execution time.
does it makes sense to move login to beforeAll()?

Reply:
Well. There are indeed a couple of reasons for it.
In the first place, this may help us to keep the tests as isolated as possible; it is true that now or rbac test are quite coupled together, but at some point, maybe with the keyloack or something similar, we can try to make them completely isolated, which shall be our objective in my opinion.
On the other hand, I saw this in the [docs](https://docs.redhat.com/en/documentation/red_hat_developer_hub/1.3/html/authorization/enabling-and-giving-access-to-rbac#enabling-and-giving-access-to-rbac) about the admin account:
Sign out from the existing Red Hat Developer Hub session and log in again using the declared policy administrator account. 
Apparently, it regards only about admin account setup, but I think it may make sense to logout and login again when making changes on RBAC elements, to ensure the changes are correctly pulled into the frontend.

Reply:
Okay got it. thanks